### PR TITLE
Fix around language

### PR DESCRIPTION
**Pull request by maiko3tattun**
- Fix text was not binding to resources
- Fixed a bug that caused the language to run in the same language as InstalledUICulture the first time it was launched, but reset to en-US when the preference window was opened

### CODE REVIEW

File: OpenUtau/App.axaml.cs [hunk 1]
```diff
@@ -59,6 +60,11 @@ public class App : Application {
             if (resDict != null) {
                 Current.Resources.MergedDictionaries.Remove(resDict);
                 Current.Resources.MergedDictionaries.Add(resDict);
+                var langName = resDict.Source.OriginalString.Replace("/Strings/Strings.", "").Replace(".axaml", "");
```
Comment (stakira) [inline OpenUtau/App.axaml.cs:63]: It is not meant to set the reference automatically. It is meant to follow system language if never manually set.

Comment (stakira) [reply OpenUtau/App.axaml.cs:63]: OK never mind. This is not unreasonable either.

Comment (maiko3tattun) [reply OpenUtau/App.axaml.cs:63]: Should I not save?
If I delete line 66, Language in the preference file will remain string.Empty.

Comment (maiko3tattun) [reply OpenUtau/App.axaml.cs:63]: I wasn't sure if I should put it in Preferences.cs, but I couldn't write the code that references the resource dictionaries.

**Pull request merged**